netbeans.apache.org/src/content/kb/docs/web/ajax-quickstart.asciidoc

```diff
@@ -55,7 +56,7 @@ version 7.x or 8.x
 ====
 * The Java EE installation of the IDE enables you to optionally install the GlassFish server and the Apache Tomcat servlet container.
 * This tutorial assumes that you have a working knowledge of the various technologies it employs (i.e., HTML, CSS, JavaScript, Java and JSP). It attempts to provide an overview of the functionality provided by the code, but _does not_ explain how the code works on a line-to-line basis.
-* If you need to compare your project with a working solution, you can link:https://netbeans.org/projects/samples/downloads/download/Samples%252FJavaScript%252FMyAjaxApp.zip[+download the sample application+].
+* If you need to compare your project with a working solution, you can link:http://web.archive.org/web/20101218014131/http://netbeans.org/project_downloads/samples/Samples/JavaScript/MyAjaxApp.zip[+download the sample application+].
```

Comment:
nitpick: not sure what to think about (ab)using the web archive as file host :) We should try to find a better solution for that IMO. Although its probably ok for this particular case since the zip is tiny and the traffic would be also low.